docs/self-managed/components/orchestration-cluster/zeebe/exporters/elasticsearch-exporter.md

```diff
@@ -12,6 +12,11 @@ import TabItem from "@theme/TabItem";
 Please refer to [supported environments](/reference/supported-environments.md#camunda-8-self-managed) to find out which versions of Elasticsearch are supported in a Camunda 8 Self-Managed setup.
 :::
 
+:::note
+As of **8.8**, Camunda uses the **Camunda Exporter** after all **≤8.7** records have been exported and consumed.  
```

Comment:
do we have a place to link to here for users to read about the Camunda Exporter?